FirstApp/API/Recipe.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Recipe.swift
+//  FirstApp
+//
+//  Created by Akua Afrane-Okese on 2020/02/18.
+//  Copyright © 2020 Akua Afrane-Okese. All rights reserved.
+//
+
+import Foundation
+
+struct SwiftForFoodResults: Decodable {
+    var resultCount: [FoodDetails]
+
+    enum CodingKeys: String, CodingKey {
+        case resultCount = "results"
+    }
+}
```

Comment:
Might I suggest that this be called FoodResults instead of SwiftForFoodResults. The SwiftFor part doesn't make much sense

Reply:
Good point, I will make the change👍.